app/models/spree/user_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::User.class_eval do
+  def full_name
```

Comment:
Is anything calling this? I would prefer not to add methods to `Spree::User` if possible.